### PR TITLE
Added inviter and date in the member invitation card (#5600)

### PR DESCRIPTION
**Pull request by Appsens**
Resolve [ #5600](https://github.com/opencollective/opencollective/issues/5600)

API: https://github.com/opencollective/opencollective-api/pull/7615

# Description

Added a message with the inviter and date in the member invitation card.

# Screenshots

image

### CODE REVIEW

File: pages/member-invitations.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ const memberInvitationsPageQuery = gqlV2`
   query MemberInvitationsPage($memberAccount: AccountReferenceInput!) {
     memberInvitations(memberAccount: $memberAccount) {
       id
+      inviter {
```
Comment (znarf) [inline pages/member-invitations.js:19]: @Betree according to your latest message, we should add `id` there.

**Review by Betree — APPROVED** [review]
Looks great!

File: pages/member-invitations.js (hunk 1 above)
Comment (Appsens) [reply pages/member-invitations.js:19]: Added id, slug and type.

**Review by Betree — COMMENTED** [review]
There seem to be many unrelated changes [in the diff](https://github.com/opencollective/opencollective-frontend/pull/7862/files), can we rebase the PR to make things clearer?

**Pull request closed without merging**